code/src/AlbumCard.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react'
+import {Icons} from './Icons'
```

Comment:
You don't need the brackets when importing only one thing 
```suggestion
import Icons from './Icons'
```